dbms/src/Flash/Pipeline/Schedule/Tasks/Task.cpp

```diff
@@ -77,48 +78,57 @@ Task::Task(MemoryTrackerPtr mem_tracker_, const String & req_id)
     GET_METRIC(tiflash_pipeline_task_change_to_status, type_to_init).Increment();
 }
 
-ExecTaskStatus Task::execute() noexcept
+Task::~Task()
+{
+    RUNTIME_ASSERT(
```

Comment:
```suggestion
    RUNTIME_CHECK(
```

Reply:
Because the destructor function in C++11 is noexcept, throwing an error can also lead to crashes and `Terminate called without an active exception`, so here we use `RUNTIME_ASSERT`.